test/integration/PermissionTable.test.ts

```diff
@@ -72,7 +72,7 @@ const table: [string, string, AM[], AM[] | undefined, string, string, number, nu
 
   [ 'PUT',     'C/',  [],                     undefined,              '',     N3,  401, 401 ],
   [ 'PUT',     'C/',  [ AM.read ],            undefined,              '',     N3,  401, 401 ],
-  [ 'PUT',     'C/',  [ AM.write ],           undefined,              '',     N3,  205, 201 ],
+  [ 'PUT',     'C/',  [ AM.write ],           undefined,              '',     '',  409, 201 ],
```

Comment:
Since we deviate from the linked table here we should add a comment.

`// We return a 409 when targeting an existing container as we only allow changes targeting the metadata directly`